bazel/glog.bzl

```diff
@@ -92,6 +92,8 @@ def glog_library(namespace = "google", with_gflags = 1, **kwargs):
     darwin_only_copts = [
         # For stacktrace.
         "-DHAVE_DLADDR",
+        "-DHAVE_PTHREAD_THREADID_NP",
```

Comment:
Could you add a comment `# Avoid deprecated syscall()`? Otherwise someone might think this is needed for a stacktrace.